package.json

```diff
@@ -2,7 +2,7 @@
   "name": "poa-dapps-voting",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://poanetwork.github.io/",
+  "homepage": "https://voting.poa.network",
```

Comment:
well, there is no reason to change it because it is useful to have it deployed on both: netlify and GH  pages

Reply:
ok, I will return back this change